pkg/utils/utils.go

```diff
@@ -52,6 +56,32 @@ func AreSelectorsSame(lhs, rhs map[string]string) bool {
 	return true
 }
 
+// ForceLinuxNodeSelector takes given selector map and returns a selector map with linux node selector added into it.
+// If there is already a node type selector and is different from "linux" then it is overridden to "linux" type.
+// See https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#interlude-built-in-node-labels
+func ForceLinuxNodeSelector(selectors map[string]string) map[string]string {
```

Comment:
instead of `ForceLinuxNodeSelector` can we call it something like `AddLinuxNodeSelector` or `SetLinuxNodeSelector`?

Reply:
I also think `EnsureLinuxNodeSelector` could make sense here